extension/src/extension.ts

```diff
@@ -13,22 +13,22 @@ import { registerTaskManager } from './taskManager';
 import { registerBuildFileWatcher } from './buildFileWatcher';
 import { Api } from './api';
 
-const localize = nls.loadMessageBundle();
+// const localize = nls.loadMessageBundle();
 
 export async function activate(context: vscode.ExtensionContext): Promise<Api> {
   const statusBarItem = vscode.window.createStatusBarItem();
-  statusBarItem.command = 'gradle.showProcessMessage';
-  statusBarItem.text = localize(
-    'extension.starting',
-    '{0} Gradle: Starting',
-    '$(sync~spin)'
-  );
-  statusBarItem.show();
+  // statusBarItem.command = 'gradle.showProcessMessage';
```

Comment:
consider adding this back in for immediate feedback when activating the extension